src/garage/torch/value_functions/base.py

```diff
@@ -0,0 +1,71 @@
+"""Base class for all baselines."""
+import abc
+
+
+class ValueFunction(abc.ABC):
+    """Base class for all baselines.
+
+    Args:
+        env_spec (garage.envs.env_spec.EnvSpec): Environment specification.
+        name (str): Value function name, also the variable scope.
+
+    """
+
+    def __init__(self, env_spec, name):
+        self._mdp_spec = env_spec
+        self.name = name
+
+    @abc.abstractmethod
+    def get_param_values(self):
```

Comment:
I'm not quite sure we need these methods right now. Let's try not having them, and we can add them later if needed.

Reply:
fixed